src/js/core/svg.js

```
@@ -1,188 +1,137 @@
+function parseSVG(svg, icon) {
+    if (icon && includes(svg, '<symbol')) {
+        svg = parseSymbols(svg)[icon] || svg;
     }
 
-};
+    return fragment(svg);
+}
 
-const symbolRe = /<symbol(.*?id=(['"])(.*?)\2[^]*?<\/)symbol>/g;
-const symbols = {};
+const symbolRe = /<symbol([^]*?id=(['"])(.+?)\2[^]*?<\/)symbol>/g;
```

Biome

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)